cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -64,9 +64,13 @@
    */
   public enum HiveSupport {
     // CDH 5.0 to 5.1 uses Hive 0.12
-    // CDH >5.1 uses Hive >=0.13.1 (aka 1.0, which Hive14ExploreService supports)
+    // The order of the enum values below is very important
```

Comment:
Probably move this to the top, otherwise the `CDH 5.0 to 5.1 uses Hive 0.12` comment may be confusing.